pkg/app/mutate_app_test.go

```diff
@@ -26,13 +27,49 @@ import (
 func Test_MutateApp(t *testing.T) {
 	ctx := context.Background()
 
+	eggs2Cluster1910 := capiv1beta1.Cluster{
```

Comment:
I don't think this cluster is tested against currently

Reply:
Required by legacy tests. Still, it will be useful for multiple Cluster test.